demos/age_gender_recognition/python/README.md

```diff
@@ -15,29 +15,35 @@ Start OVMS container with image pulled in previous step and mount `model` direct
 docker run --rm -d -u $(id -u):$(id -g) -v $(pwd)/model:/models/age_gender -p 9000:9000 -p 8000:8000 openvino/model_server:latest --model_path /models/age_gender --model_name age_gender --port 9000 --rest_port 8000
 ```
 
+On Windows open another command window and run
```

Comment:
if you change `ovms.exe` to `ovms`, it could be a common command for baremetal on windows and linux

Reply:
Yes, for bare metal isntruction it would be great, but in contrast with docker commands I would leave ovms.exe as then the reader would promptly identify the os for the command.